be/src/storage/lake/async_delta_writer.cpp

```diff
@@ -212,7 +212,7 @@ inline void AsyncDeltaWriterImpl::finish(Callback cb) {
     // by the submitted tasks.
     if (int r = bthread::execution_queue_execute(_queue_id, task); r != 0) {
         LOG(WARNING) << "Fail to execution_queue_execute: " << r;
-        task.cb(Status::InternalError("AsyncDeltaWriterImpl not open()ed or has been close()ed"));
+        task.cb(Status::InternalError("AsyncDeltaWriterImpl not open()ed or has been closed"));
```

Comment:
```suggestion
        task.cb(Status::InternalError("AsyncDeltaWriterImpl not opened or has been closed"));
```

not related to your PR, but better change together.

Reply:
Updated